nixos/modules/tasks/filesystems/zfs.nix

```
@@ -533,6 +542,13 @@ in
   config = mkMerge [
     (mkIf cfgZfs.enabled {
       assertions = [
+        {
+          assertion = cfgZfs.modulePackage.version == cfgZfs.package.version;
+          message = ''
+            The kernel module and the userspace tooling versions
+            are not matching, this is an unsupported usecase.
+          '';
```

```suggestion
          message = "The kernel module and the userspace tooling versions are not matching, this is an unsupported usecase.";
```
not need to span this over two lines in the terminal